.github/workflows/msys2_build.yml

```diff
@@ -16,10 +14,13 @@ jobs:
   build:
     runs-on: windows-latest
     strategy:
+      fail-fast: false
       matrix:
         sys:
           - mingw64
           - ucrt64
+        imtl: [main, v23.04]
+        dpdk: [21.05, 21.08, 21.11, 22.03, 22.07, 22.11, 23.03]
```

Comment:
39 checks with this change, too many. Why we need support so many DPDK version? I guess 22.11 and 23.03 is enough

Reply:
The DPDK patches before 22.11 are not for msys2 environment, CI won't pass.

Reply:
Hi,

So this is define based on the IMTL patches version which is available on main, maint-22.09, maint-22.12 & maint-22.12
Hence is range result from DPDK version 21.05 to 23.03

Hence the intention here is to cover for backward as well as current DPDK compatibility to ensure the health of IMTL libs
I do agree 39 checks maybe a little too lengthy given need to cover for MinGW64, URCT64.

DPDK 22.11 and 23.03 is only covered for the mainline (main branch)
https://github.com/OpenVisualCloud/Media-Transport-Library/tree/main/patches/dpdk

Reply:
I agree we should check the build against other version. One possible way is enable a new CI job with weekly/daily schedule, my point is no need to check old DPDK status for every PR/commit.